main.py

```diff
@@ -41,9 +45,46 @@ def image_generator(bbox_list, time_interval, evalscript, maxcc):
             yield data
 
 
+def inspect_raster_data(raster_bytes: bytes):
+    # Create a temporary file to write the bytes
+    with tempfile.NamedTemporaryFile(suffix=".tif", delete=False) as tmpfile:
+        tmpfile.write(raster_bytes)
+        tmp_filepath = tmpfile.name
+
+    # Use rasterio to open the temporary file
+    with rasterio.open(tmp_filepath, "r") as src:
+        # Read the raster data
+        data = src.read(
+            1
+        )  # Read the first band. Change the index if you have multiple bands.
+
+    # Inspect the data values
+    print(data)
+
+    # Optionally, delete the temporary file after use
+    os.remove(tmp_filepath)
+
+
+def round_to_nearest_5(input_raster: gdal.Dataset) -> gdal.Dataset:
+    band = input_raster.GetRasterBand(1)  # Assuming a single band raster
+    data = band.ReadAsArray() * 100
```

Comment:
I get at this line: 
ImportError: cannot import name '_gdal_array' from 'osgeo' (/Users/marc.leerink/dev/PlasticDetectionService/venv/lib/python3.11/site-packages/osgeo/__init__.py)

Probably a gdal version issue as well